app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+skip_before_action :login_required 
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.find_by(email: params[:session][:email].downcase)
+    if @user && @user.authenticate(params[:session][:password])
+      session[:user_id] = @user.id
+      redirect_to user_path(@user.id)
+    else
+      @user = User.new
+      flash.now[:danger] = 'ログインに失敗しました'
+      render "new"
```

Comment:
Symbol形式で記載できるものはSymbolで記載した方が良いです。
```suggestion
      render :new
```